core/consensus/aura/src/lib.rs

```diff
@@ -734,15 +735,26 @@ mod tests {
 			})
 		}
 
-		fn peer(&self, i: usize) -> &Peer<Self::PeerData> {
+		fn add_peer(&mut self, config: &ProtocolConfig) {
+			let client = Arc::new(test_client::new());
+			let verifier = self.make_verifier(client.clone(), config);
+			let (block_import, justification_import, data) = self.make_block_import(client.clone());
+			let specialization = DummySpecialization {};
```

Comment:
braces aren't necessary here, are they?

Reply:
I'm getting the below when I remove then:
```
error[E0423]: expected value, found struct `DummySpecialization`
   --> core/network/src/test/mod.rs:729:24
    |
729 |         let specialization = DummySpecialization;
    |                              ^^^^^^^^^^^^^^^^^^^ did you mean `DummySpecialization { /* fields */ }`?
```

Reply:
probably the error traces back to the struct definition then and we could address it there.

Reply:
You're right, the empty brackets had to be removed from the struct definition...